.dotrun.json

```diff
@@ -1 +1 @@
e": "2.0.4", "source-map": "0.5.7", "postcss-reporter": "7.0.1", "postcss-selector-parser": "6.0.4", "path-exists": "3.0.0", "bail": "1.0.5", "forever-agent": "0.6.1", "verbalize": "0.1.2", "amdefine": "1.0.1", "delegates": "1.0.0", "is-alphanumerical": "1.0.4", "read-pkg-up": "1.0.1", "vanilla-framework": "2.19.1", "parse-json": "5.1.0", "mkdirp": "0.5.5", "spdx-expression-parse": "3.0.1", "hard-rejection": "2.1.0", "strip-ansi": "6.0.0", "is-hexadecimal": "1.0.4", "isobject": "2.1.0", "write-file-atomic": "3.0.3", "character-entities-html4": "1.1.4", "has-color": "0.1.7", "qs": "6.5.2", "bcrypt-pbkdf": "1.0.2", "code-point-at": "1.1.0", "ansi-styles": "4.3.0", "to-regex-range": "5.0.1", "isarray": "1.0.0", "punycode": "2.1.1", "is-number": "7.0.0", "globule": "1.3.2", "error-ex": "1.3.2", "nan": "2.14.2", "dependency-graph": "0.9.0", "currently-unhandled": "0.4.1", "tough-cookie": "2.5.0", "locate-path": "3.0.0", "postcss-scss": "2.1.1", "are-we-there-yet": "1.1.5", "unist-util-visit-parents": "3.1.1", "gensync": "1.0.0-beta.1", "core-util-is": "1.0.2", "delayed-stream": "1.0.0", "os-tmpdir": "1.0.2", "read-pkg": "4.0.1", "fstream": "1.0.12", "ajv": "6.12.6", "yaml": "1.10.0", "array-find-index": "1.0.2", "readdirp": "3.5.0", "semver": "5.7.1", "unist-util-stringify-position": "2.0.3", "callsites": "3.1.0", "extend": "3.0.2", "cross-spawn": "3.0.1", "safe-buffer": "5.1.2", "postcss-value-parser": "4.1.0", "lodash.forown": "4.4.0", "is-decimal": "1.0.4", "globals": "11.12.0", "is-plain-obj": "1.1.0", "get-caller-file": "2.0.5", "pify": "2.3.0", "yargs-parser": "13.1.2", "combined-stream": "1.0.8", "is-utf8": "0.2.1", "asn1": "0.2.4", "uuid": "3.4.0", "lru-cache": "2.7.3", "util-deprecate": "1.0.2", "tar": "2.2.2", "fast-glob": "3.2.4", "ccount": "1.0.5", "ms": "2.1.2", "validate-npm-package-license": "3.0.4", "gonzales-pe": "4.3.0", "node-gyp": "3.8.0", "postcss-safe-parser": "4.0.2", "is-extglob": "2.1.1", "tslib": "1.14.1"}, "lockfile_hash": "7ae1cea010e4ab8a601b092820134ed6"}, "python": {"requirements": "canonicalwebteam.flask-base==0.6.4\n", "installed_packages": ["talisker-0.19.0.dist-info", "wheel-0.36.2.dist-info", "greenlet-0.4.16.dist-info", "urllib3-1.26.4.dist-info", "chardet-4.0.0.dist-info", "raven-6.10.0.dist-info", "requests-2.25.1.dist-info", "gunicorn-20.1.0.dist-info", "yamlloader-1.0.0.dist-info", "canonicalwebteam.yaml_responses-1.1.1.dist-info", "Flask-1.1.2.dist-info", "gevent-20.6.2.dist-info", "pip-21.0.1.dist-info", "Jinja2-2.11.3.dist-info", "canonicalwebteam.flask_base-0.6.4.dist-info", "certifi-2020.12.5.dist-info", "immutables-0.15.dist-info", "prometheus_client-0.7.1.dist-info", "PyYAML-5.4.1.dist-info", "idna-2.10.dist-info", "zope.event-4.5.0.dist-info", "setuptools-54.2.0.dist-info", "future-0.16.0.dist-info", "statsd-3.3.0.dist-info", "blinker-1.4.dist-info", "contextvars-2.4.dist-info", "itsdangerous-1.1.0.dist-info", "zope.interface-5.4.0.dist-info", "MarkupSafe-1.1.1.dist-info", "Werkzeug-0.15.6.dist-info"]}}
```

Comment:
do you mind deleting this file and add it in the `.gitignore` ?

Reply:
yep!